pilot/pkg/serviceregistry/kube/controller/authorization.go

```diff
@@ -566,7 +579,16 @@ func (c *Controller) AuthorizationPolicyHandler(old config.Config, obj config.Co
 		}
 	}
 
-	updates := c.ambientIndex.CalculateUpdatedWorkloads(pods, workloadEntries, c)
+	// 3. only process service entries in config cluster with endpoints
+	serviceEntries := []*apiv1alpha3.ServiceEntry{}
+	if c.configCluster {
+		serviceEntries = append(serviceEntries, c.getServiceEntriesInPolicy(obj.Namespace, sel)...)
```

Comment:
This seems not right, it is using the authz' workload selector to select se by matching the SE's labels, however we actualluy need to select its endpoints by matching endpoint's labels

Reply:
What if no labels on ep? Should it fallback to se?

Reply:
Updated impl is incorrect and should map a SE to set of WE